Setup.hs

```diff
@@ -1,2 +0,0 @@
-import           Distribution.Simple
-main = defaultMain
```

Comment:
I have no idea why this is still included with project templates -_-

Reply:
me neither, so i deleted it